web/app/view/ReportController.js

```diff
@@ -208,15 +210,45 @@ Ext.define('Traccar.view.ReportController', {
                 params: {
                     id: positionIds
                 },
+                scope: this,
                 callback: function (records, operation, success) {
                     if (success) {
                         Ext.getStore('ReportRoute').add(records);
+                        if (this.singleEvent && records.length > 0) {
```

Comment:
If we select first position for every event report, then we have to select event in report grid for consistent.
Also if we select everything (in three panels) it should repeat behavior of manual selection and move map to selected position. It is not good, it should fit all positions after  generating report.